hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/pipeline/ClosePipelineSubcommand.java

```diff
@@ -59,12 +59,22 @@ public void execute(ScmClient scmClient) throws IOException {
 
       List<Pipeline> pipelineList = new ArrayList<>();
       Predicate<? super Pipeline> predicate = replicationFilter.orElse(null);
-      for (Pipeline pipeline : scmClient.listPipelines()) {
-        boolean filterPassed = (predicate != null) && predicate.test(pipeline);
-        if (pipeline.getPipelineState() != Pipeline.PipelineState.CLOSED && filterPassed) {
-          pipelineList.add(pipeline);
+      List<Pipeline> pipelines = scmClient.listPipelines();
```

Comment:
I suggest to fix a condition without adding `if/else` block:
```
boolean filterPassed = (predicate == null) || predicate.test(pipeline);
```

I checked, tests are working